src/au3audio/CMakeLists.txt

```diff
@@ -28,6 +28,6 @@ include(${CMAKE_CURRENT_LIST_DIR}/../au3wrap/au3defs.cmake)
 
 set(MODULE_INCLUDE ${AU3_INCLUDE})
 set(MODULE_DEF ${AU3_DEF})
-set(MODULE_LINK au3wrap)
+set(MODULE_LINK au3wrap trackedit effects_base)
```

Comment:
just in case, if we use data types from another module that are described only in the `.h` file, or interfaces, then we do not need to link these modules.
In the current implementation of AU4, it is usually necessary to link only au3wrap and global (linked automatically)